reflex/app_mixins/lifespan.py

```diff
@@ -26,6 +27,7 @@ async def _run_lifespan_tasks(self, app: FastAPI):
         try:
             async with contextlib.AsyncExitStack() as stack:
                 for task in self.lifespan_tasks:
+                    console.debug(f"Running lifespan task: {task}")
```

Comment:
for some reason i dont see this one showing up on my end... maybe the loglevel isn't propagated into the subprocess?

Reply:
Seems so. Will investigate on that problem.

Reply:
follow up: bug as been present since forever, and is unrelated to the changes in this PR, so will need a PR of its own to resolve the issue.